scripts/micro_benchmarks.py

```diff
@@ -127,6 +127,15 @@ def __get_bdn_arguments(user_input: str) -> list:
             return args
         return []
 
+    parser.add_argument(
+        '--wasm',
+        dest='wasm',
+        required=False,
+        default=False,
+        action='store_true',
+        help='''Tests should be run with the wasm runtime'''
```

Comment:
nit: extra quotes

Reply:
That is the python feature for a string literal.